packages/dashboard/package.json

```diff
@@ -32,11 +28,20 @@
     "@types/enzyme": "3.1.5",
     "@types/enzyme-adapter-react-16": "1.0.1",
     "@types/jest": "23.0.0",
+    "@types/react": "16.3.16",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/react-dom": "16.0.5",
     "@types/webpack-env": "1.13.0",
+    "@uifabric/jest-serializer-merge-styles": ">=6.0.3 <7.0.0",
     "enzyme": "^3.2.0",
     "enzyme-adapter-react-16": "^1.1.0",
     "office-ui-fabric-react-tslint": ">=6.0.0 <7.0.0",
-    "@types/react-test-renderer": "^16.0.0",
-    "@uifabric/jest-serializer-merge-styles": ">=6.0.3 <7.0.0"
+    "react": ">=16.3.2-0 <17.0.0",
+    "react-test-renderer": "^16.2.0",
+    "react-dom": ">=16.3.2-0 <17.0.0"
+  },
+  "peerDependencies": {
+    "react": "^0.14.9 || ^15.0.1-0 || ^16.0.0-0",
```

Comment:
This package depends on oufr@6, which has a peer dep of react 16. So the peer dep range support here for react 0.x/15.x is superfluous.